### ISSUE TITLE
WPML - SELLER DASHBOARD - STORE SETTINGS > SHIPPING : many fields cannot be translatable

### ISSUE THREAD

**Issue by Sophie-2e** [getdokan/dokan-wpml#35]
in WPML - SELLER DASHBOARD - STORE SETTINGS > SHIPPING TAB: many fields cannot be translatable

List below:

- [ ] zone name
- [ ] regions
- [ ] shipping methods
- [ ] zone name field
- [ ] shippong methods fields
- [ ] edit button
- [ ] when add shipping policiy > can t translate back to zone list button
- [ ] when add shipping policiy > all form fields trasnlated but shon in english
- [ ] when go in a zone > various fields not translated

**Comment by Sophie-2e** [comment]
Any news? Still not translated and noticed even more errors regarding these shipping module translations

**Comment by Sophie-2e** [comment]
@nurul-umbhiya @shohag121  Any update on this?

**Comment by shohag121** [comment]
Hello @Sophie-2e,

We have already handled those translation issues on previous releases.

Would you please clear the cache and check again?

In the meantime, I'll inform our testing team to thoroughly check the Shipping Page translation functionalities.

Thank you for informing the issues to us.

cc: @StalinDurjo @shashwatahalder01

**Comment by Sophie-2e** [comment]
Hello @shohag121 yes indeed, I think I replied to you on https://github.com/getdokan/dokan-wpml/pull/44 for the shipping statut, all good, but all the fields mentioned in this github #35 are still not, even after clearing cache as the strings are not foundable in Dokan strings to be translated, and even with Dokan Lite and Pro 3.10.3 updates. 

Here proof wth the  zone name name title field after Dokan Lite and Pro 3.10.3 updates. :

![Screenshot 2024-04-18 at 17 38 30](https://github.com/getdokan/dokan-wpml/assets/131655229/32f2ffdb-21f4-45b9-beb4-4a77a4df88a2)

[cc](cc: @StalinDurjo @shashwatahalder01)

**Comment by Sophie-2e** [comment]
Hello @shohag121 is there any progress on this?

cc @@nurul-umbhiya @imtiaz-pranto

**Comment by MdAsifHossainNadim** [comment]
@Sophie-2e Dokan vendor shipping translation is working fine with WPML. Kindly check this video & re-check it in your system.

https://github.com/getdokan/dokan-wpml/assets/90011088/90d59122-b702-4455-849d-213f8c19861c

**Comment by Sophie-2e** [comment]
@MdAsifHossainNadim No, I have all the plugins mentionned and impossible to find these strings, still with Dokan 3.11.0.

Here all strings I cannot find with complete proof that I cannot find any of them whatever I do: https://share.vidyard.com/watch/USzot7kJtF6uJbyXuVR9om?

Waiting for your news.

**Comment by Sophie-2e** [comment]
Still not working last Dokan updates. Please do a test with WPML and then start to resolve the issue. It has been opened for more than 6 months. The Shipping Module is the worst part of Dokan so I need sellers to have it in a language they can understand to make a little bit easier. 

Thanks for your understanding, will be waiting for news.

cc @nurul-umbhiya @shohag121 @MdAsifHossainNadim

**Comment by Sophie-2e** [comment]
Hello @MdAsifHossainNadim, even with Dokan 3.11.4 and Pro 3.11.3, I am still unable to find and translate these strings with WPML, still having issue with this: https://github.com/getdokan/dokan-wpml/issues/62

Please help.

cc: @nurul-umbhiya @shohag121

**Comment by Mohaiminulislam1989** [comment]
@MdAsifHossainNadim please provide update here.

**Comment by Sophie-2e** [comment]
@MdAsifHossainNadim it has been 4 months that I am waiting for a reply. Please help is needed here. It still does not work on my side with WPML...

cc: @Mohaiminulislam1989 @shohag121 @imtiaz-pranto

**Comment by MdAsifHossainNadim** [comment]
@Sophie-2e The shipping fields WPML string translation issue is working fine on my end (without loco translators). Kindly review the [screencast](https://streamable.com/zx86wh).

**Comment by Sophie-2e** [comment]
@MdAsifHossainNadim your screencast is about the field that are translatable, and yes me too I can translate them.

But here is fields missing list: https://github.com/getdokan/dokan-wpml/issues/35#issue-2076166155
Video: https://share.vidyard.com/watch/XHrAyLAvMnQbRZnS9SnDNA?

Your team has access to the staging website so you can see yourself. Please tell me how to fix it now. Thanks.

**Comment by J-H-Mojumder** [comment]
It would be nice to have an update on this.

CC: @MdAsifHossainNadim

**Comment by Sophie-2e** [comment]
Any news on this @MdAsifHossainNadim? If there are no bugs as you said, would love to know how it can be fixed on my website (knowing that the strings do not show up in WPML strings even after rescanning all plugsin and theme)? And if there are bugs, what are the next steps to get it fixed?

cc: @J-H-Mojumder @imtiaz-pranto

**Comment by Sophie-2e** [comment]
@MdAsifHossainNadim @J-H-Mojumder @imtiaz-pranto 

Any update to resolve this issue?

**Comment by osmansufy** [comment]
Related PR:
https://github.com/getdokan/dokan/pull/2377
https://github.com/getdokan/dokan-pro/pull/3612

**Comment by MdAsifHossainNadim** [comment]
Zone name fields translation, handled by `WooCommerce` & Policy dynamic instruction fields not being translated.
Rest of the translation will be workable on [Shipping API and React implementation](https://github.com/getdokan/dokan-pro/pull/3997) PR.

**Comment by jamil-mahmud** [comment]
> Zone name fields translation, handled by `WooCommerce` & Policy dynamic instruction fields not being translated. Rest of the translation will be workable on [Shipping API and React implementation](https://github.com/getdokan/dokan-pro/pull/3997) PR.

@MdAsifHossainNadim vai, as per your reply kindly have a discussion with Mahbub vai that the issue can be closed or not?

**Comment by jamil-mahmud** [comment]
@MdAsifHossainNadim vai, did you have any discussion on this?
